MovieApp/MovieApp/Model/MovieManager.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  MovieManager.swift
+//  MovieApp
+//
+//  Created by Julio Gabriel Tobares on 14/04/2022.
+//
+
+import Foundation
+
+protocol MovieManagerDelegate {
+    func onNowLoaded(movie: Result<[Movie], CustomError>)
+    func onPopularLoaded(movie: Result<[Movie], CustomError>)
```

Comment:
Don't use 'Result<>' with delegate. You are already handling Result cases in Manager. If, and only if, you use the data sent in delegate, use '[Movie]' instead and change parameter to 'movies'. If you don't use data sent in delegate, don't have parameters

Reply:
Argument of each func deleted